apps/server/src/middleware/security.ts

```
@@ -0,0 +1,86 @@
+					`script-src 'self' 'nonce-${nonce}' 'strict-dynamic' https:; ` +
+					`style-src 'self' https:; ` +
+					`img-src 'self' data: blob: https:; ` +
+					`font-src 'self' https: data:; ` +
+					`connect-src 'self' https: wss:; ` +
+					`media-src 'self' https: data:; ` +
+					`object-src 'none'; ` +
+					`base-uri 'self'; ` +
+					`form-action 'self'; ` +
+					`frame-ancestors 'none'; ` +
+					`upgrade-insecure-requests; ` +
+					`block-all-mixed-content;`
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)